README.md

```diff
@@ -1,77 +1,373 @@
+
+Following will be the directions to operate in the command line for MacOS and Windows.
+
+### Command Line Operations 
+
+#### Open
+You can use the Terminal app to access your computer's command line. The Terminal is one of the most powerful tools on your Mac, and it's how we'll be running our code.
+
+MacOS: To open Terminal, go to Applications, then to Utilities finally to Terminal.
+Windows: To open the command prompt, click Start and then click All Programs. In the programs list, click Accessories. In the accessories list, click Command Prompt.
+
+#### Print Working Directory 
+
+To help you know where you are currently in the Terminal we need to print the wordking directory. A directory would look like a folder on the screen of the GUI( Graphical User Interface). Type this command and hit enter:
 
-Below is a tutorial covering what web development is, what web development is comprised of, as well as how it all comes together to create
-web applications. The tutorial mentions installing Atom, don't worry about that. You're going to use VScode instead. 😉
+MacOS: `pwd`
+Windows: `cd`
 
-[HTML & CSS Tutorial (Go Through The Tutorial)](https://internetingishard.com/html-and-css/)
+#### List Directories
+To list all the directories and files in your current location type:
+MacOS : `ls`
+Windows: `dir`
 
-Getting a good grasp of how to layout websites is VERY important. The course below is going to get you moving on just that with some cool new additions to CSS.
+#### Make Directory
 
-<h2 id="javascript">JavaScript</h2>
+We want a place where we can put all our work in. While in the terminal make a new directory called VetsWhoCode by typing mkdir in Windows or MacOS and press enter:
 
-JavaScript is where you go from being an artist to being a wizard in web development. A deep knowledge of JavaScript fundamentals will make you a highly employable developer. Complete Javascript Info to get a good beginner grasp of JS.
+`mkdir VetsWhoCode`
 
-JavaScript is derived from ECMAScript. This is important since new versions come out having major impact on JavaScript. Read through this resource to understand the newest version of ECMAScript:
+#### Change Working Directory
 
+Now that we have made a new directory lets go there where we will put our work on our machine. te do that by in both windows and MacOS we do this by dyping cd then pressing enter:
 
+`cd VetsWhoCode`
 
-[Javascript Info](http://javascript.info/)
 
+#### Change to previous Directory
+If you want to go back to the previous directory for both Windows and MacOS you type this and press enter:
 
+`cd ..`
 
-<h2 id="capstone">Capstone</h2>
+#### Delete a directory
+Let's say you make a typo and need to delete a directory. Here are the commands to do that.
```

Comment:
> make a typo

add, `make a typo in a folder name`

Also, please add a warning that `rm -rf` should not be taken lightly. You have the ability to delete important files that could screw up your computer. If this makes you nervous, it's okay to open the file explorer and right-click delete a folder.